### ISSUE TITLE
Task: Update docs to reflect minimal mode

### ISSUE THREAD

**Issue by austinlparker** [open-telemetry/opentelemetry-demo#970]
With #872 merged, we have several docs updates that need to be done. See https://github.com/open-telemetry/opentelemetry-demo/pull/872/#issuecomment-1627605516 for details.

**Comment by Caritajoe18** [comment]
Hello @austinlparker, I would like to be assigned this issue to update the document. Can you kindly provide more context on the project? Thank you.

**Comment by puckpuck** [comment]
@Caritajoe18 the demo can be run with fewer components deployed, consuming significantly less memory. I can't remember all the changes so that should be documented. This is the [make target](https://github.com/open-telemetry/opentelemetry-demo/blob/main/Makefile#L139-L148) that runs it. You can compare docker-compose.yml and docker-compose-minimal.yml to see all the differences.

**Comment by vitorvasc** [comment]
Hey @open-telemetry/demo-approvers, we've merged https://github.com/open-telemetry/opentelemetry.io/pull/9779. Is this issue still valid, or can it be closed?

**Comment by osullivandonal** [comment]
> Hey [@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers), we've merged [open-telemetry/opentelemetry.io#9779](https://github.com/open-telemetry/opentelemetry.io/pull/9779). Is this issue still valid, or can it be closed?

Just looking at [opentelemetry.io](https://opentelemetry.io/docs/demo/docker-deployment/) there seems to be some formatting issues witht the new minimal section:

<img width="1435" height="1137" alt="Image" src="https://github.com/user-attachments/assets/e0c75d62-e1c1-4c6b-a816-d6fa2fdbff0c" />

Can we address this before we close this?

**Comment by vitorvasc** [comment]
@osullivandonal, absolutely, thanks for the finding! Just raised a new issue to fix the broken behavior: https://github.com/open-telemetry/opentelemetry.io/issues/9914
